### ISSUE TITLE
[Web UI] TypeError: Cannot read property 'current' of undefined

### ISSUE THREAD

**Issue by schu** [sensu/sensu-go#1841]
**Web UI Stack Trace:**

```
at `undefined` ([src/components/controller/MenuController.js:27](https://github.com/sensu/sensu-go/blob/2d9b937/dashboard/src/components/controller/MenuController.js#L27))
at `ModalController.render` ([src/components/controller/ModalController.js:27](https://github.com/sensu/sensu-go/blob/2d9b937/dashboard/src/components/controller/ModalController.js#L27))
at `l` (node_modules/react-dom/cjs/react-dom.production.min.js:131)
at `beginWork` (node_modules/react-dom/cjs/react-dom.production.min.js:138)
at `e` (node_modules/react-dom/cjs/react-dom.production.min.js:176)
at `f` (node_modules/react-dom/cjs/react-dom.production.min.js:176)
at `G` (node_modules/react-dom/cjs/react-dom.production.min.js:182)
at `x` (node_modules/react-dom/cjs/react-dom.production.min.js:181)
at `n` (node_modules/react-dom/cjs/react-dom.production.min.js:181)
at `B` (node_modules/react-dom/cjs/react-dom.production.min.js:180)
at `v` (node_modules/react-dom/cjs/react-dom.production.min.js:179)
at `b` (node_modules/react-dom/cjs/react-dom.production.min.js:190)
at `Object.updateContainer` (node_modules/react-dom/cjs/react-dom.production.min.js:191)
at `tg.render` (node_modules/react-dom/cjs/react-dom.production.min.js:228)
at `undefined` (node_modules/react-dom/cjs/react-dom.production.min.js:242)
```

Component Stack:

```
in ModalController
in MenuController
in ButtonMenu
in div
in ButtonSet
in WithStyles
in div
in Typography
in WithStyles
in TableListHeader
in WithStyles
in EventsListHeader
in WithStyles
in div
in div
in Loader
in WithStyles
in div
in Paper
in WithStyles
in ListController
in EventsContainer
in u
in withApollo(EventsContainer)
in div
in AppContent
in WithStyles
in Query
in Query
in EventsContent
in QueryParams
in Route
in withRouter(QueryParams)
in withQueryParams(EventsContent)
in Route
in Switch
in div
in div
in div
in Loader
in WithStyles
in AppFrame
in WithStyles
in Query
in Query
in AppWrapper
in EnvironmentView
in Route
in ConditionalRoute
in AuthenticatedRoute
in Query
in Apollo(AuthenticatedRoute)
in Switch
in MuiThemeProvider
in ThemeProvider
in AppThemeProvider
in Connect(AppThemeProvider)
in ApolloProvider
in Provider
in AppRoot
in Route
in withRouter(AppRoot)
in WithStyles
in Router
in BrowserRouter
in ErrorBoundary
```

**Comment by jamesdphillips** [comment]
@schu thanks for reporting! The issue should resolved by #1842.

**Comment by schu** [comment]
@jamesdphillips thanks for the swift fix :+1:
